app/src/main/java/com/kickstarter/ui/extensions/SafeLet.kt

```diff
@@ -0,0 +1,24 @@
+package com.kickstarter.ui.extensions
+
+inline fun <T1 : Any, T2 : Any, R : Any> safeLet(p1: T1?, p2: T2?, block: (T1, T2) -> R?): R? {
```

Comment:
I love this! but a question, could we user varargs to reduce the amount of `safeLet` declarations?

Reply:
I am not sure, but I'll see what I can find!

Reply:
from what i can tell I can do a vararg, but the problem is that seems to make it so all the elements need to be the same type, whereas this allows each thing being passed to be whatever it wants to be as long as its not null.  ill keep digging on that when i can, but for now I will push this up